packages/pixel-demo/test/swingsetTests/gallery/vat-alice.js

```diff
@@ -103,7 +103,10 @@ function makeAliceMaker(E, log, contractHost) {
 
           const bobsRawPixel = result.extent[0];
           // TODO this was previously unchecked
-          console.log(`UNCHECKED ASSERTION NOW CHECKED HERE: ${bobsRawPixel.x === rawPixel.x && bobsRawPixel.y === rawPixel.y}`);
```

Comment:
Note, that this assertion is like this because it actually fails. This was the only silently failing one that actually failed. As it turns out, the assertion appears wrong rather than the surrounding code..

Reply:
Now that this is gone, I don't know what it is about. If irrelevant, please resolve.

Reply:
Nevermind. I was looking at the wrong vat-alice.js